src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,7 +106,10 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        LocalDate today = LocalDate.now();
+        return (today.isAfter(someDate)) ? someDate + " is before " + today
+                : (today.isBefore(someDate)) ? someDate + " is after " + today
+                : someDate + " is today";
```

Comment:
let's use relevant methods, for example someDate.isAfter(today) ? someDate + " is after " + today...